apps/cli-daemon/src/app/workspace-manager/workspace-manager.controller.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { InternalServerErrorException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { WorkspaceManagerController } from './workspace-manager.controller';
+import { WorkspaceManagerService } from './workspace-manager.service';
+
+type WorkspaceManagerServiceMock = Record<
+  keyof WorkspaceManagerService,
+  jest.Mock
+>;
+const workspaceManagerMock: WorkspaceManagerServiceMock = {
+  getDirectoriesInPath: jest.fn(),
+};
+
+describe('WorkspaceManagerController', () => {
+  let controller: WorkspaceManagerController;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [WorkspaceManagerController],
+      providers: [
+        { provide: WorkspaceManagerService, useValue: workspaceManagerMock },
+      ],
+    }).compile();
+
+    controller = module.get<WorkspaceManagerController>(
+      WorkspaceManagerController
+    );
+  });
+
+  it('should be defined', () => {
+    expect(controller).toBeDefined();
+  });
+
+  describe('getDirectory', () => {
+    it('Should call workspaceManagerService.getDirectoriesInPath', async () => {
+      await controller.getDirectory('abc');
+      expect(workspaceManagerMock.getDirectoriesInPath).toHaveBeenCalledWith(
+        'abc'
+      );
+    });
+    it('Should throw InternalServerErrorException when workspaceManagerService.getDirectoriesInPath throws error', async () => {
```

Comment:
```suggestion
    it('should throw InternalServerErrorException when workspaceManagerService.getDirectoriesInPath throws error', async () => {
```

Reply:
you have some more "Should" just change them all to "should"

Reply:
done :)